### PR TITLE
fix issues to k8 service being fetched in case of knative resources

### PR DESCRIPTION
**Pull request by invincibleJai**
fix issues to k8 service being fetched in case of knative resources

Tracks: https://jira.coreos.com/browse/ODC-2159

### CODE REVIEW

File: frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts [hunk 1]
```diff
@@ -99,10 +99,8 @@ const createKnativeDeploymentItems = (
     }
     return overviewItems;
   }
-  const services = transformResourceData.getServicesForResource(resource);
-  const routes = transformResourceData.getRoutesForServices(services);
   const knResources = getKnativeServiceData(resource, resources);
-  return { obj: resource, buildConfigs: [], routes, services, ...knResources };
+  return { obj: resource, buildConfigs: [], routes: [], services: [], ...knResources };
```
Comment (joshuawilson) [inline frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts]: This looks like you drop all services and get none. When I run this, it looks like `routes` and `services` are undefined. 
If that is the case, then you might need to filter the `resources` to get the knative services out.

Comment (invincibleJai) [reply frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts]: we are already fetching knative specific resources i.e configurations, routes, revisions etc in https://github.com/openshift/console/pull/3212/files#diff-6710cf4682ac7597af9d6f5e58a10860R102

 ```
const knResources = getKnativeServiceData(resource, resources);
```

**Pull request merged**